net.jockx.Kulki.yml

```diff
@@ -0,0 +1,24 @@
+id: net.jockx.Kulki
```

Comment:
Can't access https://jockx.net/

![image](https://github.com/flathub/flathub/assets/62639087/9eb02e76-2123-4bca-8959-04e9973d15b2)

Reply:
Ah it is https://kulki.jockx.net/